radio/src/gui/colorlcd/model_setup.cpp

```diff
@@ -1222,6 +1222,32 @@ void ModelSetupPage::build(FormWindow * window)
     grid.addWindow(new TrainerModuleWindow(window, {0, grid.getWindowHeight(), LCD_W, 0}));
   }
 
+//OW
+#if defined(TELEMETRY_MAVLINK)
```

Comment:
I think we should place this on the telemetry page, where I would have expected it. Any reasons for not doing so?